gluten-data/src/main/scala/io/glutenproject/execution/GlutenRowToArrowColumnarExec.scala

```diff
@@ -60,12 +60,12 @@ object RowToColumnConverter {
       case StringType => StringConverter
       case BinaryType => BinaryConverter
       case CalendarIntervalType => CalendarConverter
-      case at: ArrayType => new ArrayConverter(getConverterForType(at.elementType, nullable))
+      case at: ArrayType => new ArrayConverter(getConverterForType(at.elementType, at.containsNull))
       case st: StructType => new StructConverter(st.fields.map(
         (f) => getConverterForType(f.dataType, f.nullable)))
       case dt: DecimalType => new DecimalConverter(dt)
       case mt: MapType => new MapConverter(getConverterForType(mt.keyType, nullable),
```

Comment:
The MapType key nullable check is right?

Reply:
Key can't be null, just set false